gempak/ush/gdas_ukmet_meta_ver.sh

```diff
@@ -226,22 +139,28 @@ EOFplt
     done
 done
 
-export err=$?;err_chk
+export err=$?
+
 #####################################################
 # GEMPAK DOES NOT ALWAYS HAVE A NON ZERO RETURN CODE
 # WHEN IT CAN NOT PRODUCE THE DESIRED GRID.  CHECK
 # FOR THIS CASE HERE.
 #####################################################
-ls -l ukmetver_12.meta
-export err=$?;export pgm="GEMPAK CHECK FILE";err_chk
-
-if [ $SENDCOM = "YES" ] ; then
-    mkdir -p -m 775 ${COMOUTukmet}/ukmet.${PDY}/meta/
-    mv ukmetver_12.meta ${COMOUTukmet}/ukmet.${PDY}/meta/ukmetver_${PDY}_12
-    if [ $SENDDBN = "YES" ] ; then
-        ${DBNROOT}/bin/dbn_alert MODEL UKMETVER_HPCMETAFILE $job \
-        ${COMOUTukmet}/ukmet.${PDY}/meta/ukmetver_${PDY}_12
-    fi
+if (( err != 0 )) || [[ ! -s ukmetver_12.meta ]]; then
+    echo "FATAL ERROR: Failed to create ecmwf meta file"
+    exit "${err}"
+fi
+
+mv ukmetver_12.meta "${COM_ATMOS_GEMPAK_META}/ukmetver_${PDY}_12"
```

Comment:
Similar to the ecmwf meta file, there is no ukmetver_20240310_12 file under /lfs/h1/ops/prod/com/gfs/v16.3/gdas.20240310/00/atmos/gempak/meta.

Reply:
This file is generated for the 06 cycle only. It is located at /lfs/h1/ops/prod/com/gfs/v16.3/gdas.20240315/06/atmos/gempak/meta/ukmet.20240315/meta.